tests/suites/deploy_aks/deploy_aks.sh

```diff
@@ -21,6 +21,11 @@ run_deploy_aks_charms() {
 
 	echo "Destroy model"
 	juju destroy-model "test-deploy-aks-charms" --destroy-storage -y
+
```

Comment:
maybe we could add a `FIXME()...` or a `NOTE(...` so we can find it later?

Reply:
I've added TODO message.